games/s.yaml

```
@@ -272,6 +272,23 @@
   type: remake
   updated: 2017-04-27
 
+- name: Serious-Sam-Android
+  info: 'A port of Serious Sam: the Second Encounter to Android. There is also a fork
+  for the Pandora.'
+  lang:
+  - C++
+  - C
+  license:
+  - Custom
```

```suggestion
  - GPL2
```
from https://github.com/aarcangeli/Serious-Sam-Android/blob/master/Serious-Engine/LICENSE